### ISSUE TITLE
fastapi suport video-display serving?

### ISSUE THREAD

**Issue by yuanjie-ai** [tiangolo/fastapi#1994]
fastapi suport video-display serving?

**Comment by ycd** [comment]
Yes it does. You are looking for [**StreamingResponse**](https://fastapi.tiangolo.com/advanced/custom-response/#streamingresponse).

**Comment by tiangolo** [comment]
Thanks for the help here @ycd ! :clap: :bow: 

If that solves the original question, then you can close this issue @Jie-Yuan  :heavy_check_mark:

Otherwise, please follow the issue template step by step, that way it's easier to help.
